charts/kargo/templates/management-controller/cluster-roles.yaml

```diff
@@ -36,6 +36,7 @@ rules:
   - kargo.akuity.io
   resources:
   - projects
+  - projectconfigs
```

Comment:
Looks like @hiddeco added projectconfigs to these rules recently, with a narrower set of permissions down on line 54/56. You can remove this here and just add get, list, and watch down there. Update and delete shouldn't be needed.